pkg/operator/encryption/controllers/migrators/metrics.go

```diff
@@ -0,0 +1,70 @@
+package migrators
+
+import (
+	k8smetrics "k8s.io/component-base/metrics"
+	"k8s.io/component-base/metrics/legacyregistry"
+)
+
+const (
+	namespace = "storage_migrator"
+	subsystem = "core_migrator"
+)
+
+// metrics provides access to all core migrator metrics.
+var metrics *migratorMetrics
+
+func init() {
+	metrics = newMigratorMetrics()
+}
+
+// migratorMetrics instruments core migrator with prometheus metrics.
+type migratorMetrics struct {
+	objectsMigrated *k8smetrics.CounterVec
+	migration       *k8smetrics.CounterVec
+}
+
+// newMigratorMetrics create a new MigratorMetrics, configured with default metric names.
+func newMigratorMetrics() *migratorMetrics {
+	objectsMigrated := k8smetrics.NewCounterVec(
+		&k8smetrics.CounterOpts{
+			Namespace: namespace,
+			Subsystem: subsystem,
+			Name:      "migrated_objects",
```

Comment:
I think that this should have `_total` suffix (we could try to upstream that change)